types/eventConfig/requirements.ts

```diff
@@ -183,3 +183,13 @@ export const SignUpActionSchema = z.object({
  * Sign up action type
  */
 export type SignUpAction = z.infer<typeof SignUpActionSchema>;
+
+/**
+ * REALMS VOTE ACTION
+ */
+export const RealmsVoteActionSchema = z.object({
+  daoPubKey: z.string(),
+  proposalPubKey: z.string(),
+  curstomProgramId: z.string().optional(),
```

Comment:
Just spelling of `curstomProgramId` here

Also, should this field be added in client?

Reply:
cusrtom lol ... yes eventually but I dont wanna add it until we support that... does that make senes?